web/src/stream_popover.js

```diff
@@ -566,9 +570,62 @@ export async function build_move_topic_to_stream_popover(
         event.stopPropagation();
     }
 
+    function caseInsensitiveInlucde(topics, current_topic) {
```

Comment:
This function name doesn't follow our naming conventions and has a typo in its name; can you fix that?